content/01.abstract.md

```diff
@@ -1,10 +1,13 @@
 ## Abstract {.page_break_before}
 
-Important tasks in biology and biomedicine such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
-Degree strongly influences edge prediction, and degree biases and degree imbalance can lead to nonspecific predictions.
-We introduce a network permutation framework to quantify the effect of node degree on network-based methods and prediction tasks.
-As part of this we define an edge prior to quantify the probability that two nodes are connected based only on their degree.
-After demonstrating that this prior feature shows excellent discrimination and calibration performance for 20 different biomedical networks (16 bipartite, 3 undirected, 1 directed), we conclude that our prior feature represents a suitable baseline for network link prediction tasks, as performance exceeding the baseline is attributable to factors other than degree alone.
-Additionally, we propose methods to incorporate network permutation and the edge prior into other predictive methods.
-Our results highlight the importance of degree for link prediction and provide software to account for its effects when degree bias may be present.
-We have released a full implementation of our network permutation method and the edge prior as an open-source Python package on GitHub and the Python Package Index.
+Important tasks in biomedical discovery such as predicting gene functions, gene-disease associations, and drug repurposing opportunities are often framed as network edge prediction.
+The number of edges connecting to a node, termed degree, can vary greatly across nodes in real biomedical networks.
+If degree strongly influences edge prediction, then imbalance or bias in the distribution of degrees could lead to nonspecific or misleading predictions.
+We introduce a network permutation framework to quantify the effects of node degree on edge prediction.
+Our framwork decomposes performance into the proportion attributable to degree and the proportion attributable to edge-dependent inference.
+Additionally, our permutation-derived edge prior quantifies the probability of a connection based only on node degree.
+This feature shows excellent discrimination and calibration for 20 biomedical networks (16 bipartite, 3 undirected, 1 directed), with AUROCs frequently exceeding 0.85.
+Feature performance exceeding the edge prior is attributable to factors other than degree and is often only a small portion of overall performance.
```

Comment:
```suggestion
Feature performance exceeding the edge prior is attributable to factors other than degree, and we find that is often only a small portion of overall performance.
```

Reply:
I tried to emphasize this a bit more. I agree with @dhimmel that this is the most important point.